src/components/Tooltip/Tooltip.tsx

```diff
@@ -19,7 +23,7 @@ const Content = styled(RadixTooltip.Content)<{ $width: TooltipProps['width'] }>`
   animation-timing-function: cubic-bezier(0.16, 1, 0.3, 1);
   will-change: transform, opacity;
   width: ${pipe(prop('$width'), pxToRem)};
-  z-index: 1;
+  z-index: ${({ $isInFloatingElement }) => ($isInFloatingElement ? 1300 : 1)};
```

Comment:
shouldn't we use the `getDepth` helper method?

Reply:
yeah we probably should